jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -642,6 +635,36 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Wrapper that generates a string based on various state to send to GA.
+
+      @method sendAnalytics
+      @param {arguments} arguments All arguments passed will be used.
+    */
+    sendAnalytics: function() {
```

Comment:
With these changes you remove the category ex) `Button Click` and convert it into part of the action, was this intentional? If not you probably want to modify this signature to be:

```JavaScript
sendAnalytics: function(category, ...args) {
```